### PR TITLE
fix(gcs): Prevent NPE when gcs artifact account has no credentials path

### PR DESCRIPTION
**Pull request by sbwsg**

### CODE REVIEW

File: clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/gcs/GcsArtifactCredentials.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.storage.Storage;
 import com.google.api.services.storage.StorageScopes;
+import org.apache.commons.lang.StringUtils;
```
Comment (duftler) [inline clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/gcs/GcsArtifactCredentials.java]: Sort order is off here.

**Comment by sbwsg** [comment]
This doesn't fix the specific bug I'm looking into unfortunately.  Just couldn't get clouddriver to start without adding this.

File: clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/gcs/GcsArtifactCredentials.java (hunk 1 above)
Comment (sbwsg) [reply clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/gcs/GcsArtifactCredentials.java]: Ah, java noob question: Which part of the import name should I be sorting on?  I've just been doing alphabetical, so "org" comes after "com"

Comment (lwander) [reply clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/gcs/GcsArtifactCredentials.java]: That's right, but you also have "com" entries below this.

Comment (sbwsg) [reply clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/gcs/GcsArtifactCredentials.java]: doh

**Pull request merged**